src/shogun/features/DenseFeatures.cpp

```diff
@@ -829,7 +829,15 @@ SGVector<ST> CDenseFeatures<ST>::mean() const
 }
 
 template <typename ST>
-SGVector<float64_t > CDenseFeatures<ST>::std(bool colwise) const
+ST CDenseFeatures<ST>::median() const
+{
+	ASSERT_FLOATING_POINT
```

Comment:
do you need this, median of features rather than the matrix?

Reply:
I wanted to add it, but then I realised that the iterators go along each training example, rather than column, so it became a bit of a pain without copying data...

Reply:
removed as I ended up just using linalg

Reply:
ok makes more sense imo